jupyterhub/templates/hub/configmap.yaml

```diff
@@ -38,6 +38,11 @@ data:
   {{- end }}
   {{- end }}
 
+  {{ if eq .Values.auth.type "custom" -}}
+  auth.custom.class-name: {{ .Values.auth.custom.className | quote }}
+  auth.custom.traitlets: {{ toJson .Values.auth.custom.traitlets | quote }}
```

Comment:
Yep, done.

Unrelated question, since I haven't used github pull requests before. Is it normal to pile up commits like this? Are you going to squash them on your end when you merge, or are you just going to leave all of them separate.